libs/src/api/file.js

```diff
@@ -11,6 +11,11 @@ const publicGateways = [
   'https://cloudflare-ipfs.com/ipfs/'
 ]
 
+/**
+ * Download url if body document exists.
```

Comment:
@vicky-g this is technically not what this function does.  it's a cool way to get a file to download through javascript using an element in the DOM